components/clp-package-utils/clp_package_utils/general.py

```
@@ -132,59 +123,66 @@ def generate_container_name(job_type: str) -> str:
     return f"clp-{job_type}-{str(uuid.uuid4())[-4:]}"
 
 
-def check_dependencies():
+def check_docker_dependencies(should_compose_run: bool, project_name: str):
```

Ruff (0.14.0)

Boolean-typed positional argument in function definition

(FBT001)